test/test-code/2-game-slug-release.js

```diff
@@ -0,0 +1,240 @@
+import {
+  gameReleasesURL,
+  browser,
+  createUserGroupGameRelease,
+  login,
+  sleep,
+  Release
+} from '../helpers';
+import { expect } from 'chai';
+import { until, By, error, WebElement } from 'selenium-webdriver';
+/**
+ *
+ * @param {0 | 1 | 2} permission
+ * @param {0 | 1 | 2} privilege
+ * @param {"dev" | "qa" | "stage" | "prod"} gameStatus
+ */
+export const basic = async (permission, privilege, gameStatus) => {
+  const { user, game } = await createUserGroupGameRelease({
+    permission,
+    privilege,
+    gameStatus
+  });
+  const url = gameReleasesURL(game);
+  await login(user);
+
+  await browser.get(url);
+
+  const err = await browser
+    .wait(until.elementsLocated(By.css('[data-target="#addRelease"]')), 500)
+    .catch(err => err);
+
+  expect(err).to.not.be.instanceOf(error.TimeoutError);
+
+  const text = await browser.findElement(By.className('panel-title')).getText();
+
+  expect(text).to.equal('Releases');
+  return game;
+};
+
+export const publicUserTest = async () => {
+  const { game } = await createUserGroupGameRelease();
+  const url = gameReleasesURL(game);
+  await browser.get(url);
+
+  await browser.wait(until.elementLocated(By.className('form-login')), 1000);
+};
+
+/**
+ *
+ * @param {boolean} pass
+ */
+export const changeTest = async pass => {
+  const formButton = await browser.findElement(
+    By.className(`btn btn-default btn-block status dropdown-toggle`)
+  );
+
+  expect(await formButton.isEnabled()).to.equal(pass);
+
+  if (pass) {
+    const oldText = await formButton.getText();
+
+    await formButton.click();
+
+    const radio = await browser.findElement(
+      By.css(
+        '.dropdown-menu.statusChange-menu > li:not(.active) > a > input[type="radio"]'
+      )
+    );
+
+    await radio.click();
+
+    const newText = await browser
+      .findElement(
+        By.className(`btn btn-default btn-block status dropdown-toggle`)
+      )
+      .then(async ele => await ele.getText());
+
+    if (pass) {
+      expect(newText).to.not.equal(oldText);
+    } else {
+      expect(newText).to.equal(oldText);
+    }
+  }
+};
+
+export const downloadTest = async () => {
+  await browser.findElement(By.className('help-block updated')).click();
+
+  await browser
+    .findElement(
+      By.css(
+        'body > div > div > div.col-sm-9 > div > div.panel-body.releases > div:nth-child(1) > div > div.col-sm-9 > div > div > form:nth-child(2) > div > button'
```

Comment:
I feel that this style of selector, although specific, is going to make the tests brittle... I'd almost rather add an class to the element in question and select by that. That way, we can refactor the UI without breaking the page

Reply:
I'd prefer that, but most of pages are build in a way that targeting a specific element is impossible without writing selectors like this. This one though can be simplified as I just copied the path from chrome.